pyproject.toml

```diff
@@ -52,6 +51,9 @@ inflection = "^0.5.1"
 sqlalchemy = "^1.4"
 python-dotenv = ">=0.20,<0.22"
 typing-extensions = "^4.2.0"
+simplejson = "^3.17.6"
+jsonschema = "^4.16.0"
+pytz = "^2022.2.1"
```

Comment:
These were dependencies of `pipelinewise-singer-python`, so I'm adding them explicitly here.